src/main/kotlin/org/rust/ide/utils/import/ImportCandidatesCollector2.kt

```diff
@@ -398,6 +396,7 @@ private fun ImportContext2.createPathWithImportAdded(usePath: String): RsPath? {
 }
 
 private fun QualifiedNamedItem2.toImportInfo(defMap: CrateDefMap, modData: ModData, needExternCrate: Boolean): ImportInfo {
+    val path = path.map2Array(String::escapeIdentifierIfNeeded)
```

Comment:
Do it really work a expected?
It looks like new implementation changes `defMap.hasExternCrateInCrateRoot(crateName)` and `crateName in modData.childModules` behavior since `crateName` is escaped now. But I haven't checked it in practice